scripts/open-coredump.sh

```diff
@@ -360,9 +360,11 @@ fi
 if [ "$(grep -o ~dev <<< $VERSION)" == "~dev" ]
 then
     BRANCH=${MAIN_BRANCH}
+    JENKINS_WORKSPACE_DIR=/jenkins/workspace/scylla-${MAIN_BRANCH}/next
 else
     BASE_VERSION=$(grep -o "^[0-9]\+\.[0-9]\+" <<< $VERSION)
     BRANCH=branch-${BASE_VERSION}
+    JENKINS_WORKSPACE_DIR=/jenkins/workspace/scylla-${BASE_VERSION}/build
```

Comment:
As I recall, cores from production builds, used to work out-of-the-box. Do we really need this substitution?
It doesn't too much, because if a substitutation rule doesn't match, GDB will ignore it, but it is noise and can be confusing.

Reply:
I see. I needed this substitution rule for debugging cores from the promotion run. How about printing this command only for non-release builds? What do you think?